Frontend/ui-driver/src/Screens/HomeScreen/View.purs

```diff
@@ -1126,3 +1128,11 @@ getDriverStatusResult index driverStatus currentStatus= case (getValueToLocalSto
                             else DEFAULT
                   _ -> if (driverStatus == currentStatus) then ACTIVE
                        else DEFAULT
+
+launchMaps :: forall action. (action -> Effect Unit) ->  action -> Flow GlobalState Unit
+launchMaps push action = do
+  void $ delay $ Milliseconds 2000.0
```

Comment:
can you move the delay as well within the condition?

Reply:
No, there is a case where if he clicks on MAPS button after the condition is executed, in this case google maps will be launched two time

Reply:
if he clicks within 2 seconds?